smartmin/views.py

```diff
@@ -420,6 +420,8 @@ class SmartTemplateView(SmartView, TemplateView):
 
 
 class SmartReadView(SmartView, DetailView):
+    slug_field = None
+    slug_url_kwarg = None
```

Comment:
You mean introduce our own field like `use_slug`? That would mean subclasses have to define `slug_url_kwarg = 'uuid'`, `slug_field = 'uuid'` and `use_slug = True` to use a slug field like UUID. 

Anyways since our default case doesn't support `slug_field` and `slug_url_kwarg` (i.e. get_url_pattern sets up a pk based URL), seems fitting to nullify them